examples/mission-control/Cargo.toml

```diff
@@ -25,3 +26,20 @@ opt-level = "z"
 lto = true
 debug = false
 panic = "abort"
+
+[package.metadata.near.build]
+# This is the Docker metadata line:
+image = "nearprotocol/contract-builder:latest-amd64"
```

Comment:
these 3 lines and their copies in other files look to have come out of nowhere and like they're not used.
Please remove them unless you can explain how they're supposed to be used or are used.